action/main.py

```diff
@@ -14,9 +14,10 @@
 
 run([sys.executable, "-m", "venv", str(ENV_PATH)], check=True)
 
-req = "black[colorama]"
-if VERSION:
-    req += f"=={VERSION}"
+version_specifier = VERSION
+if VERSION and VERSION[0] in "0123456789":
```

Comment:
I could do `VERSION[0].isnumeric()` here though that's rather wide due to it supporting all Unicode characters that use `Numeric_Type` class. Personally, I tend to avoid isdecimal/isdigit/isnumeric methods because they evaluate to True for more than I actually want but I can change this.

Reply:
I guess it's fine either way, since we only care about recognising version specifier characters, which won't be decimal or digits. But no biggie.

Reply:
Do packaging tools normalize non-ascii digits? If so, I'd prefer `isdecimal()` otherwise I'd prefer being stricter.

Reply:
`pip` doesn't handle Unicode numbers:
```
❯ pip install 'black==२२.१'
ERROR: Could not find a version that satisfies the requirement black==२२.१ (from versions: 18.3a0, 18.3a1, 18.3a2, 18.3a3, 18.3a4, 18.4a0, 18.4a1, 18.4a2, 18.4a3, 18.4a4, 18.5b0, 18.5b1, 18.6b0, 18.6b1, 18.6b2, 18.6b3, 18.6b4, 18.9b0, 19.3b0, 19.10b0, 20.8b0, 20.8b1, 21.4b0, 21.4b1, 21.4b2, 21.5b0, 21.5b1, 21.5b2, 21.6b0, 21.7b0, 21.8b0, 21.9b0, 21.10b0, 21.11b0, 21.11b1, 21.12b0, 22.1.0, 22.3.0, 22.6.0, 22.8.0)
ERROR: No matching distribution found for black==२२.१
```

So in the end, it shouldn't matter whether we use `isdecimal()` or the `in` check as it will fail anyway.